packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -21,8 +21,10 @@ import {
   useVisit,
   navigate,
   useConfig,
+  showModal,
+  // getGlobalStore,
```

Comment:
Please don't leave commented-out code handing around. Either uncomment it or remove it...

Reply:
I was trying to see if I can detect form updates from the global store but that didn't seem to work as expected, I will remove this implementation now.